netpyne/analysis.py

```diff
@@ -596,91 +615,87 @@ def plotRaster (include = ['allCells'], timeRange = None, maxSpikes = 1e8, order
         - syncLines (True|False): calculate synchorny measure and plot vertical lines for each spike to evidence synchrony (default: False)
         - lw (integer): Line width for each spike (default: 2)
         - marker (char): Marker for each spike (default: '|')
-        - popColors (dict): Dictionary with color (value) used for each population (key) (default: None)
+        - popColors (odict): Dictionary with color (value) used for each population (key) (default: None)
```

Comment:
@jjcarre why odict?

Reply:
I didn't make that change, but there is nothing in the old or new plotRaster that enforces an odict